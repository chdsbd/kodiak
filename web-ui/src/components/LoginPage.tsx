import React from "react"
import { docsUrl, helpUrl, installUrl, loginUrl } from "../settings"

export function LoginPage() {
  return (
    <div className="h-100 d-flex justify-content-center align-items-center">
      <div
        className="w-100 text-center d-flex justify-content-around align-items-center flex-column"
        style={{ minHeight: 300 }}>
        <div className="d-flex justify-content-center align-items-center">
          <img
            src="/favicon.ico"
            alt="favicon"
            height={30}
            width={30}
            className="mr-2"></img>
          <h1 className="h2 mb-0 font-weight-bold">Kodiak</h1>
        </div>

        <div>
          <a href={loginUrl} className="gh-install-btn">
            Login with GitHub
          </a>
        </div>

        <p className="mb-0">
          <a href={installUrl}>Install</a> | <a href={docsUrl}>Docs</a> |{" "}
          <a href={helpUrl}>Help</a>
        </p>
      </div>
    </div>
  )
}
