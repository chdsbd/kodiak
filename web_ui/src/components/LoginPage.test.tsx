import React from "react"
import { render } from "@testing-library/react"
import { LoginPage } from "./LoginPage"

describe("LoginPage", () => {
  test("snap smoke test", () => {
    const { container } = render(<LoginPage />)

    expect(container).toMatchSnapshot()
  })
})
