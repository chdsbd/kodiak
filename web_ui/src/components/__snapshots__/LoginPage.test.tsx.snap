// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage snap smoke test 1`] = `
<div>
  <div
    class="h-100 d-flex justify-content-center align-items-center"
  >
    <div
      class="w-100 text-center d-flex justify-content-around align-items-center flex-column"
      style="min-height: 300px;"
    >
      <div
        class="d-flex justify-content-center align-items-center"
      >
        <img
          alt="favicon"
          class="mr-2"
          height="30"
          src="/favicon.ico"
          width="30"
        />
        <h1
          class="h2 mb-0 font-weight-bold"
        >
          Kodiak
        </h1>
      </div>
      <div>
        <button
          class="gh-install-btn"
        >
          Login with GitHub
        </button>
      </div>
      <p
        class="mb-0"
      >
        <a
          href="https://github.com/marketplace/kodiakhq"
        >
          Install
        </a>
         | 
        <a
          href="https://kodiakhq.com/docs/quickstart"
        >
          Docs
        </a>
         |
         
        <a
          href="https://kodiakhq.com/help"
        >
          Help
        </a>
      </p>
    </div>
  </div>
</div>
`;
